playwright/network.py

```diff
@@ -173,3 +173,18 @@ def request(self) -> Request:
     @property
     def frame(self) -> "Frame":
         return self.request.frame
+
+
+def serialize_headers(headers: Dict[str, str]) -> List[Header]:
+    headers_list = list()
+    for name in headers:
+        header: Header = dict(name=name, value=headers[name])
+        headers_list.append(header)
+    return headers_list
+
+
+def parse_headers(headers: List[Header]) -> Dict[str, str]:
```

Comment:
same for here, but feel free to keep your solution if you think its more readable.

`return {header["name"]: header["value"] for header in headers}`

Reply:
Done